components/post_view/post_body/post_body.jsx

```diff
@@ -190,12 +199,15 @@ export default class PostBody extends React.PureComponent {
                                 apostrophe,
                             }}
                         />
-                        <a
-                            className='theme'
+                        <span
```

Comment:
Can you add `role='button'` and `tabindex='0'` to this span? That's needed to make the span properly accessible from the keyboard or when using accessibility devices since it's no longer an anchor tag

Reply:
Oh, I never thought of that.  I'll keep that in mind.
It's now updated, thanks!